frontend/src/constants/QuestionConstants.ts

```diff
@@ -7,6 +7,7 @@ const editorTooltips = {
     "Users will have to select the correct responses from a list of options",
   MULTIPLE_CHOICE:
     "Users will have to select the best response from a list of options",
+  FRACTION: "Users will be able to input fractions in their answer",
```

Comment:
I'm also thinking it would be good to update the language here slightly (change "Users" to "Students") since teachers/admins won't think of students as "users," but I'm fine to do that in a separate ticket to generally improve copy.